components/organisms/ChangePasswordUser/ChangePasswordUser.tsx

```diff
@@ -107,14 +68,37 @@ export const ChangePasswordUser = () => {
       );
     }
   };
+  const validatePasswordUser = async () => {
+    if (userInfo?.uid !== undefined) {
+      const validatePasswordData = await validatePassword(
+        userInfo?.uid,
+        currentPassword
+      );
+      if (validatePasswordData.ok) {
+        if (currentPassword === password) {
+          Swal.fire(
+            "Error de validación",
+            "La contraseña actual y la contraseña nueva deben ser diferentes",
+            "error"
+          );
+
+        } else {
+          return true;
+        }
+      } else {
+        setErrorMessage(validatePasswordData.msg);
+        return false;
+      }
+    }
+  };
 
   const validatePasswordSubmit = async (
     e: React.FormEvent<HTMLFormElement>
   ) => {
     e.preventDefault();
 
     try {
-      const isPasswordValid = await validatePassword();
+      const isPasswordValid = await validatePasswordUser();
```

Comment:
This variable should be `isValidatedUserPassword`

Reply:
changed validateUserPassword